webapp/package.json

```diff
@@ -17,34 +17,34 @@
     "npm": ">=6.4.1"
   },
   "dependencies": {
-    "@angular/animations": "^12.2.1",
-    "@angular/common": "^12.2.1",
-    "@angular/compiler": "^12.2.1",
-    "@angular/core": "^12.2.1",
-    "@angular/forms": "^12.2.1",
-    "@angular/platform-browser": "^12.2.1",
-    "@angular/platform-browser-dynamic": "^12.2.1",
-    "@angular/router": "^12.2.1",
+    "@angular/animations": "^12.2.16",
+    "@angular/common": "^12.2.16",
+    "@angular/compiler": "^12.2.16",
+    "@angular/core": "^12.2.16",
+    "@angular/forms": "^12.2.16",
+    "@angular/platform-browser": "^12.2.16",
+    "@angular/platform-browser-dynamic": "^12.2.16",
+    "@angular/router": "^12.2.16",
     "@ngrx/effects": "^11.1.1",
     "@ngrx/store": "^11.1.1",
     "@ngx-translate/core": "^13.0.0",
     "bikram-sambat": "^1.7.0",
     "bikram-sambat-bootstrap": "^1.5.0",
     "bootstrap": "^3.4.1",
     "bootstrap-daterangepicker": "^2.1.30",
-    "enketo-core": "4.41.6",
+    "enketo-core": "4.60.6",
     "eurodigit": "^3.1.3",
     "font-awesome": "^4.7.0",
-    "google-libphonenumber": "^3.2.25",
+    "google-libphonenumber": "^3.2.26",
     "gsm": "^0.1.4",
-    "jquery": "3.2.x",
+    "jquery": "^3.6.0",
```

Comment:
It needs to match with the one inside Enketo, I believe it's lower version than this one.

Reply:
fixed in 6067565b7